src/components/latest-rules-content/latestRulesContent.js

```diff
@@ -69,23 +70,77 @@ const LatestRulesContent = ({
     return rule.slice(6, rule.length);
   };
 
+  const openUserRule = async (path) => {
+    const loginName = await fetchGithubName(path);
+    const pathPrefix = process.env.NODE_ENV === 'development' ? '' : '/rules';
+
+    if (loginName) {
+      navigate(`${pathPrefix}/user-rules/?author=${loginName}`);
+    } else {
+      navigate('/404');
```

Comment:
Should we be navigating to a 404 page here?

Reply:
🤔 I'm not sure.. Is it better to show an alert?

Reply:
I think since its not an issue with the page, sending them to the user results page for them to see an empty list with no results achieves better feedback to the user. Maybe we can fix the "No Results..." text to something else, but I think its fine. 

![image](https://github.com/SSWConsulting/SSW.Rules/assets/57518417/9dc40c42-ef92-4d9a-a8be-f09cc23b3780)
**Figure: Current no results**

Reply:
Done - Removed 404 navigation. Now, whether or not there's a login name, users will be sent to the rules page. 🙂